test/unit/api/response/iiif/presentation-api/metadata.test.js

```diff
@@ -24,7 +24,7 @@ describe("IIIF response presentation API metadata helpers", () => {
   it("metadataLabelFields(source)", () => {
     const metadata = metadataLabelFields(source);
     expect(Array.isArray(metadata)).to.be;
-    expect(metadata.length).to.eq(28);
+    expect(metadata.length).to.eq(29);
```

Comment:
Ha, not sure what the purpose of this is but I'm sure it seemed like a good idea at the time.